webapp/components/sidebar_right_menu.jsx

```diff
@@ -264,6 +265,25 @@ export default class SidebarRightMenu extends React.Component {
                     </li>
                 );
             }
+
+            if (global.window.mm_config.EnableTeamCreation === 'true' || isAdmin) {
```

Comment:
I'm sorry I did a mistake, needs to be systemAdmin and not only Admin. Will change that